modules/navier_stokes/doc/content/source/functormaterials/INSFVMushyPorousFrictionFunctorMaterial.md

```diff
@@ -31,8 +31,7 @@ F = C_F \frac{\rho_l}{\sqrt{K}} \,,
 where the Ergun coefficient is $C_F = 0.55$ and $\rho_l$ is the density of the liquid phase.
 
 !alert note
-This material is compatible with [INSFVMomentumFriction.md] which multiplies the
-incoming Forchheimer coefficient by the velocity magnitude; it is *incompatible*
+This material is compatible with the previous version of the friction Kernel: INSFVMomentumFriction which multiplies the incoming Forchheimer coefficient by the velocity magnitude; it is *incompatible*
```

Comment:
```suggestion
This material is compatible with the previous version of the friction Kernel: `INSFVMomentumFriction` which multiplies the incoming Forchheimer coefficient by the velocity magnitude; it is *incompatible*
```

Reply:
Looking at the source code, it looks like it is not about the speed. The mushy kernel does not include speed for the Forchheimer coefficient so in that sense this is correct now. But it includes a density. I think it would be nice to set this straight in this PR to make sure what comes out of this kernel is correct with the friction formulations in the `PINSFVMomentumFriction`. Great cache, this was potentially a bug here before.

Reply:
I would ask @tanoret to provide the references for the mushy correlations so that you can make sure it is correct. I think dividing by the wrong value and multiplying with the right value at an input file level is not the solution here. Especially that the right value might not be the right value.

Reply:
Atthe same time, I haven't seen the references so maybe the Forchheimer term does not need a factor and then feel free to disregard this.

Reply:
This is not resolved.

The text blatantly says this is not compatible with PINSFVMomentumFriction rn too

Reply:
I think the INSFVMushyPorousFrictionFunctorMaterial needs to be edited to become compatible with PINSFVMomentumFriction.